ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -72,19 +73,20 @@ Verify RHODS Installation
   IF  "${UPDATE_CHANNEL}" != "odh-nightlies"
        Wait For Pods Numbers  1
        ...                   namespace=${OPERATOR_NAMESPACE}
-       ...                   label_selector=name=rhods-operator
+       ...                   label_selector=name=${OPERATOR_NAME}
        ...                   timeout=2000
        Wait For Pods Status  namespace=${OPERATOR_NAMESPACE}  timeout=1200
        Log  Verified redhat-ods-operator  console=yes
   END
 
   IF  "${UPDATE_CHANNEL}" == "odh-nightlies" or "${cluster_type}" != "managed"
+      Apply DSCInitialization CustomResource    dsci_name=${DSCI_NAME}
```

Comment:
@aloganat this isn't necessary usually at least for our Jenkins jobs. Why we need it here now? Do we do something different in Jenkins? From a brief look to our code there, I don't think so :thinking:

Reply:
@aloganat @tarukumar Could you please check it once

Reply:
@jstourac yes we need to create only DSC for self managed instance. @vaibhavjainwiz Could you test this once without having this line ?

Reply:
As I check, on `RHODS` DSCI is created automatically when operator installed, where as on ODH, developer need to explicitly create the DSCI resource. I verified, my solution with RHODS and ODH both. Its work for both.






[3:35](https://redhat-internal.slack.com/archives/D06EMGEPC4S/p1712225150535939)
I verified, my solution with RHODS and ODH both. Its work for both.

Reply:
To avoid any issues in working code, I am adding an check so that new `DSCI` would only be created for ODH.

Reply:
done.. I had pushed the changes. @jstourac Could you please review it again.